satsense/features/pantex.py

```diff
@@ -22,40 +22,11 @@ def get_rii_dist_angles():
         pixels_dist_223 = np.array([[1, -2], [2, -1], [2, 1], [1, 2]])
         pixels_dist_3 = np.array([[2, 2], [2, -2]])
 
-        angles_1 = np.arctan2(pixels_dist_1[:, 0], pixels_dist_1[:, 1])
-        distances_1 = [
-            sp.spatial.distance.euclidean(
-                [0, 0], [pixels_dist_1[i, 0], pixels_dist_1[i, 1]])
-            for i in range(len(pixels_dist_1[:, 0]))
-        ]
-
-        angles_14 = np.arctan2(pixels_dist_14[:, 0], pixels_dist_14[:, 1])
-        distances_14 = [
-            sp.spatial.distance.euclidean(
-                [0, 0], [pixels_dist_14[i, 0], pixels_dist_14[i, 1]])
-            for i in range(len(pixels_dist_14[:, 0]))
-        ]
-
-        angles_2 = np.arctan2(pixels_dist_2[:, 0], pixels_dist_2[:, 1])
-        distances_2 = [
-            sp.spatial.distance.euclidean(
-                [0, 0], [pixels_dist_2[i, 0], pixels_dist_2[i, 1]])
-            for i in range(len(pixels_dist_2[:, 0]))
-        ]
-
-        angles_223 = np.arctan2(pixels_dist_223[:, 0], pixels_dist_223[:, 1])
-        distances_223 = [
-            sp.spatial.distance.euclidean(
-                [0, 0], [pixels_dist_223[i, 0], pixels_dist_223[i, 1]])
-            for i in range(len(pixels_dist_223[:, 0]))
-        ]
-
-        angles_3 = np.arctan2(pixels_dist_3[:, 0], pixels_dist_3[:, 1])
-        distances_3 = [
-            sp.spatial.distance.euclidean(
-                [0, 0], [pixels_dist_3[i, 0], pixels_dist_3[i, 1]])
-            for i in range(len(pixels_dist_3[:, 0]))
-        ]
+        (angles_1, distances_1) = __get_rii_dist_angle(pixels_dist_1)
```

Comment:
```suggestion
        (distances_1, angles_1) = _get_rii_dist_angle(pixels_dist_1)
```
since this function returns `(distance, angle)`, and also for the function calls below.